### PR TITLE
Added Play Store app url to README

### PR DESCRIPTION
**Pull request by beaufortfrancois**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 The BLE Peripheral Simulator is an Android app that allows developers to try
 out new features of Web Bluetooth without the need for a BLE Peripheral Device.
 
+You can build it from source or download it from the Google Play store at
```
Comment (scheib) [inline README.md]: "or download it" -> "or install it".

I suggest hyperlinking 'Google Play store' instead of the raw link.

**Comment by scheib** [comment]
LGTM with suggested changes.

**Comment by beaufortfrancois** [comment]
Suggested changes have been applied. Thanks for reviewing.

**Comment by beaufortfrancois** [comment]
You may want to set the Website GitHub URL to https://play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral as well.

screenshot 2015-07-07 at 4 54 57 pm

**Comment by g-ortuno** [comment]
Done. Thanks!

**Pull request merged**